ui/src/components/create_client/CreateXamarinClient.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react';
+import { Form  } from 'patternfly-react';
+import { renderForm } from './CreateClientFormUtils';
+
+/**
+ * Component for the Android specific create mobile client form.
+ */
+class CreateXamarinClient extends Component {
+    render() {
+        const formFields = [
+            {
+                controlId:'appName',
+                label:'* Application Name',
```

Comment:
Is there some recommended proper way in Patternfly (bootstrap) to mark the field required?